CONTRIBUTING.md

```diff
@@ -1,3 +1,11 @@
 # How to Contribute
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/AndcultureCode/AndcultureCode/blob/master/CONTRIBUTING.md) in the Home repo.
\ No newline at end of file
+## Considerations
+This repo is now distributing packages for both module loaded applications as well as global distrubtion for things such as websites.  What this means is:
+
+1.  When adding new peer dependencies to this repo, please make sure to handle them appropriately within [webpack.config.js](./webpack.config.js).
+2.  Please add any new needed `<script />` peer dependency references to  [sample-global-distro.html](./sample-global-distro.html).
+3.  Add a small, quick example to test that peer dependency within [sample-global-distro.html](./sample-global-distro.html).  You will need to run `npm run build` in order to regenerate the global package beforehand.  Open up the HTML file in your browser and validate there are no javascript errors.
```

Comment:
Thought: Feel like we could possibly automated test this ongoing - perhaps we create a github issue. I can see this step being missed ongoing and regressions going out. That said, gotta start some where and this documentation is very helpful 👍

Reply:
On second thought: after reading through how critical it is that references below must be a specific way, I feel quite strongly that we should have a sure fired way to ensure folks don't miss this. Otherwise, it is almost a guarantee that it will regress. Also, I'd like a sure fired strategy before we follow this pattern across our other repos.

Reply:
I'm gonna bounce some ideas off of Stef here to see about rounding this out. I know you are pretty busy this week. If you need to run with it now, I'd say bundle and publish it outside of AndcultureCode _right now_. I'll have a direction shortly!

Reply:
👍